portfolio/src/main/webapp/community.js

```diff
@@ -1,57 +1,58 @@
 function renderComments() {
-let quantity = document.getElementById("quantity").value;
-  fetch('/comments?quantity=' + quantity).then(response => response.json()).then((conversation) => {
-    
-    let commentSection = document.getElementById('submitted-comments');
-    while (commentSection.lastChild) {
-        commentSection.removeChild(commentSection.lastChild);
-    }
-
-    let commentCount = 0;
-    conversation.forEach((comment) => {
-        createComment(comment, commentCount%2);
-        commentCount++;
-    });
-  });
+  let commentRequestedNum =
+      document.getElementById('commentRequestedNum').value;
+  fetch('/comments?commentRequestedNum=' + commentRequestedNum)
+      .then(response => response.json())
+      .then((conversation) => {
+        let commentSection = document.getElementById('submitted-comments');
+        while (commentSection.lastChild) {
+          commentSection.removeChild(commentSection.lastChild);
+        }
+
+        let commentCount = 0;
+        conversation.forEach((comment) => {
+          createComment(comment, commentCount % 2);
+          commentCount++;
+        });
+      });
 }
 
 function createComment(comment, color) {
-    const commentSection = document.getElementById('submitted-comments');
-    const commentDivElement = document.createElement('div');
-    const usernameElement = document.createElement('p');
-    const commentElement = document.createElement('p');
-    const exitButtonElement = document.createElement('button');
+  const commentSection = document.getElementById('submitted-comments');
+  const commentDivElement = document.createElement('div');
+  const usernameElement = document.createElement('p');
+  const commentElement = document.createElement('p');
+  const exitButtonElement = document.createElement('button');
+
+  let commentDivClasses = new Array(2);
+  commentDivClasses[0] = 'comment-div';
 
-    let commentDivClasses = new Array(2);
-    commentDivClasses[0] = 'comment-div';
+  exitButtonElement.className = 'left';
 
-    exitButtonElement.className = 'left';
+  commentDivClasses[1] = color===0 ? 'lavender' : "lavender-blush";
```

Comment:
Formatting: add spaces around the triple equals.

Reply:
Added spaces